lib/utils/sdp-utils.js

```diff
@@ -12,6 +12,30 @@ const mergeSdpMedia = (sdp1, sdp2) => {
   return sdpTransform.write(parsedSdp1);
 };
 
+const isOpusFirst = (sdp) => {
+  const parsedSdp = sdpTransform.parse(sdp);
+  const idx = parsedSdp.media[0].rtp.findIndex((entry) => entry.codec === 'opus');
```

Comment:
since we reuse this function can we factor it out into a single line function, e.g
```
const getCodecPlacement = (sdp, codec ) => sdpTransform.parse(sdp).media[0].rtp.findIndex((e) => e.codec === codec):
```

Reply:
see my comment below, I don't understand your point